projects/m4/011-distinct-names/js/index.js

```diff
@@ -0,0 +1,60 @@
+//011
+
+//package
+const fs = require('fs');
+const process = require('process');
+
+//variable
+const folderFiles = '\\name-files-2000-200'
+
+//function read files and elaborate them
+const readMultipleFiles = () => {
```

Comment:
Why you function "read" -> console.log ?

Reply:
fixed, by giving "folderFiles" as an argument of the function